### PR TITLE
Append ApplyFunction to math functions

### PR DESCRIPTION
**Pull request by ronkok**
As described at: https://www.w3.org/TR/REC-MathML/chap3_2.html#sec3.2.4

Character U+2061 is the same as &ApplyFunction;. Ref: https://www.w3schools.com/charsets/ref_html_entities_a.asp

### CODE REVIEW

File: src/functions/operatorname.js [hunk 1]
```diff
@@ -69,7 +69,7 @@ defineFunction({
         identifier.setAttribute("mathvariant", "normal");
 
         const operator = new mathMLTree.MathNode("mo",
-            [mml.makeText("&ApplyFunction;", "text")]);
+            [mml.makeText("\u2061", "text")]);
```
Comment (edemaine) [inline src/functions/operatorname.js]: Maybe worth a comment that this is `<mo>&ApplyFunction;</mo>` as above?

Comment (ronkok) [reply src/functions/operatorname.js]: Will do.

**Comment by edemaine** [comment]
Thanks again for the PR!

**Comment by kevinbarabash** [comment]
It might be nice to have an option to control the output of MathML.  I like `&ApplyFunction;` because it's easier see what's going on, but the unicode char takes up less space which is preferable in production.

**Pull request merged**